src/configs.test.ts

```diff
@@ -0,0 +1,98 @@
+import {readdirSync} from 'fs';
+import {join} from 'path';
+
+import {Config, Variant} from './types';
+
+describe('configs', () => {
+  const configsPath = join(__dirname, 'configs');
+  const configFileNames = readdirSync(configsPath);
+
+  describe.each(configFileNames)('%s', (configFileName) => {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const config: Config = require(`configs/${configFileName}`).config;
+
+    describe.each(Object.entries(config))('property "%s"', (_, variants) => {
+      const variantsByName = variants.map(({name}, index) => [
+        name,
+        variants[index],
+      ]);
+      it.each(variantsByName)(
+        'variant "%s" uses a unique Figma name',
```

Comment:
```suggestion
        'variant "%s" should use a unique Figma name',
```

Reply:
This was my intuition as well, but this is prohibited by [the ESLint configuration for Jest](https://github.com/Shopify/web-configs/blob/aa331f2221074413a1659dc98819bbb7836d4025/packages/eslint-plugin/lib/config/jest.js#L23).

Reply:
Then let's use `must`, which isn't vague and was actually how I wrote my suggestion the first time! (I like proper usage of RFC 2119!)

Reply:
Good idea! I couldn't figure out why they would ban "should," but perhaps it is indeed to push us to use more definitive language.